spec/dummy_models_total_count_controller_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'spec_helper'
+
+describe DummyModelsTotalCountController, type: :controller do
+  describe '#index' do
+    let!(:dummy_models) { create_list(:dummy_model, 28) }
+    let(:expected_list) do
+      dummy_models.first(25).map do |dummy|
+        { 'id' => dummy.id, 'name' => dummy.name, 'something' => dummy.something }
+      end
+    end
+
+    context 'when paginating an ActiveRecord with no previous pagination but kaminari installed' do
+      before do
+        get :index
+      end
+
+      include_examples 'total count pagination param'
+
+      it 'responds with valid page' do
+        expect(response_body(response)['page']).to eq expected_list
+      end
+    end
+
+    context 'when paginating with page and limit params' do
```

Comment:
:thinking: when are you using page and limit params in the request?

Reply:
In the `dummy_models_controller` with `get :index_with_params`